src/components/StyledDivider.tsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```

Comment:
Not sure why w have the `"react/jsx-uses-react": "error",` rule enabled when the purpose of the `jsx-runtime` plugin is to disable it ...